lib/smart_city/test_data_generator.ex

```diff
@@ -107,7 +104,7 @@ defmodule SmartCity.TestDataGenerator do
     put_in(
       dataset_map,
       [:technical, :systemName],
-      "#{dataset_map.technical.orgName}__#{dataset_map.technical.dataName}"
+      "#{Faker.Color.It.name()}#{Faker.Cat.name()}__#{dataset_map.technical.dataName}"
```

Comment:
Why do we not need the orgName here anymore?

Reply:
We still do use orgName as part of systemName, but it's no longer generated as part of the dataset so we don't have access to it. We are now just using Faker to simulate a value.